src/js/stores/MesosStateStore.js

```diff
@@ -1,5 +1,6 @@
 import PluginSDK from "PluginSDK";
 import { request } from "@dcos/mesos-client";
+import Rx from "rxjs";
```

Comment:
🎨  I'd import `Observable` instead of `Rx` to reduce the amount of redundancy.

Reply:
I like how rxjs has these split up imports <3 `import { Observable } from 'rxjs/Observable';`